test-cypress/integration/setup/connectors-lucene.spec.js

```diff
@@ -0,0 +1,153 @@
+describe('Setup / Connectors - Lucene', () => {
+
+    let repositoryId;
+    const luceneConnectorName = 'lucene-test-index';
+    const fieldName = 'grape';
+    const connectorPropertyChain = 'http://www.ontotext.com/example/wine#madeFromGrape';
+    const uriType = 'http://www.ontotext.com/example/wine#Wine';
+    const connectorCreateToastMessage = 'Created connector ' + luceneConnectorName;
+    const connectorDeleteToastMessage = 'Deleted connector ' + luceneConnectorName;
+
+    before(() => {
+        repositoryId = 'repo' + Date.now();
+        cy.createRepository({id: repositoryId});
+        cy.presetRepositoryCookie(repositoryId);
+        cy.visit('/connectors');
+    });
+
+    after(() => {
+        cy.deleteRepository(repositoryId);
+    });
+
+    it('Test that no indexes are available', () => {
+        getConnectorsPage()
+            .should('be.visible')
+            .and('contain', 'No connector instances.');
+    });
+
+    it('Create a lucene connector', () => {
+        getNewLuceneConnectorButton()
+            .click();
+        getCreateLuceneConnectorPage()
+            .should('be.visible')
+            .and('contain', 'Create new Lucene Connector');
+        getConnectorNameField()
+            .find('input')
+            .type(luceneConnectorName);
+        getFieldNameField()
+            .find('input')
+            .type(fieldName, {force:true});
+        getPropertyChainField()
+            .find('input')
+            .type(connectorPropertyChain, {force:true});
+        getUriTypes()
+            .find('input')
+            .type(uriType);
+        getOkButton()
+            .click();
+        getConnectorCreationDialog()
+            .should('be.visible')
+            .and('contain', luceneConnectorName);
+        getConnectorStatusToastMessage()
+            .should('be.visible')
+            .and('contain', connectorCreateToastMessage);
+    });
+
+    it('Test connector copy functionality', () => {
+        getConnectorsPage()
+            .should('be.visible');
```

Comment:
This is already tested in the `Test that no indexes are available`, so no need to duplicate verificatons.

Reply:
Fixed with commit fe274051f6f1c19f89ac97e134e6086d2d98512c